### DISCUSSION TITLE
Consolidate ports

### DISCUSSION THREAD

**Discussion by mitchelljhirst** [directus/directus#2873] [Archive]
We are currently trying to run directus on an azure web app but do to it opening ports constantly we are having latency issues. Is there a way to set directus to reuse open connections instead of opening new ones every time.

**Comment by rijkvanzanten** [comment]
There is no way to configure that in v8. The upcoming v9 does this out of the box

**Reply by mitchelljhirst** [reply]
Okay thank you.
